client/components/recycleAndReuseComponents/VerifyItem.js

```diff
@@ -163,8 +164,19 @@ const VerifyItem = ({ items, setItems, navigateToTakeAction }) => {
                     </Button>
                 </VStack>
             </form>
+            {generalWasteItems && generalWasteItems.length > 0 && generalWasteItemDetails && <VerifyItemDialog 
+                setNextStep={navigateToTakeAction}
+                showDialog={showDialog}
+                setShowDialog={setShowDialog}
+                generalWasteItems={generalWasteItems}
+                generalWasteItemDetails={generalWasteItemDetails}
+                selectedItems={selectedItems}
+                setItems={setItems}
+                dialogItemIndex={dialogItemIndex}
+                setDialogItemIndex={setDialogItemIndex}
+            />}
         </Flex>
     )
 }
 
-export default VerifyItem;
+export default VerifyItem
```

Comment:
Basic scenarios:

1. No items selected in Add Items
    a. submit button is disabled
    b. use case ends

2. Single (blue bin/non blue bin) recyclable item selected in Add Items
    a. Item is cleaned (blue bin) / good condition (non blue bin) in Verify Items
        a1. item is displayed on final item list
        a2. user can click on self-disposal
        a3. use case ends
    b. Item is not cleaned (blue bin) / spoilt beyond repair (non blue bin) in Verify Items
        b1. item is struck off on final item list
        b2. user is redirected back to Add Items
        b3. use case ends

3. Single general waste item selected in Add Items
    a. item is shown in an alert in Verify Items
    b. item is shown with reason and suggestion in an alert after Verify Items
    c. user is redirected back to Add Items

Other more complicated scenarios can be composed from these basic scenarios. A user is only able to see the self-disposal button if they have at least one valid recyclable item (cleaned blue-bin / in good condition non-blue bin item).